miio/cooker_multi.py

```diff
@@ -0,0 +1,419 @@
+import enum
```

Comment:
Please move this and all other integration files (including tests) under `miio/integrations/chunmi/cooker_multi` like other integrations currently have it.

Reply:
done